pkg/bff/collaborators.go

```diff
@@ -78,6 +79,40 @@ func (s *Server) AddCollaborator(c *gin.Context) {
 	c.JSON(http.StatusOK, collaborator)
 }
 
+// ListCollaborators lists all the collaborators on the user's organization. The user
+// must have the read:collaborators permission to make this request.
+func (s *Server) ListCollaborators(c *gin.Context) {
+	var (
+		err          error
+		org 		*models.Organization
+	)
+
+	// Fetch the organization from the claims
+	// NOTE: This method handles the error logging and response
+	if org, err = s.OrganizationFromClaims(c); err != nil {
+		return
+	}
+
+	// Build the response from the internal map
+	out := &api.ListCollaboratorsReply{
+		Collaborators: make([]*models.Collaborator, 0),
+	}
+
+	for _, collab := range org.Collaborators {
+		if err = s.LoadCollaboratorDetails(collab); err != nil {
+			log.Error().Err(err).Str("collabID", collab.Key()).Msg("could not load collaborator details from Auth0")
+		}
+		out.Collaborators = append(out.Collaborators, collab)
+	}
+
+	// Sort by email address so we return consistent responses
```

Comment:
I'm assuming this makes the frontend work easier, however it might not be required if it's going to be re-sorted anyway. One reason I added this was to make the output for this endpoint consistently ordered since iterating over the map is non-deterministic.